starter_code/index.html

```diff
@@ -1,115 +1,210 @@
 <!DOCTYPE html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css"
+      integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2" crossorigin="anonymous">
 
     <!-- link the font awesome CDN -->
     <link
       href="https://use.fontawesome.com/releases/v5.0.1/css/all.css"
-      rel="stylesheet"
-    />
+      rel="stylesheet"/>
 
     <!-- link your styles -->
     <link rel="stylesheet" href="./stylesheets/style.css" />
     <title>Where work happens | Slack</title>
   </head>
 
   <body>
+    <!-- nav -->
     <nav>
-      <img src="./images/slack-logo.png" alt="slack_logo" />
+
+      <a href="#">
+        <img class="slack-logo" src="./images/slack-logo.png" alt="slack-logo" />
+      </a>
+
+      <img class="nav-icon" src="./images/menu.svg" alt="nav-icon">
+
       <ul>
         <li>Why Slack?</li>
         <li>Pricing</li>
         <li>About Us</li>
-        <li>Your Workspaces</li>
+        <li class="workspace">Your Workspaces</li>
       </ul>
+
     </nav>
-    <header>
-      <img src="./images/home_talking.png" alt="home_work_happening" />
-      <h1>Where Work Happens</h1>
-      <p>
-        When your team needs to kick off a project, hire a new employee, deploy
-        some code, review a sales contract, finalize next year's budget, measure
-        an A/B test, plan your next office opening, and more, Slack has you
-        covered.
-      </p>
-      <button type="button" name="button">GET STARTED</button>
-      <p>Already using Slack? <a href="#">Sign in</a></p>
+
+    <!-- header -->
+    <header class="intro">
+
+      <div class="container">
+
+        <div class="row">
+          <div class="col-lg-6">
+
+            <div class="row">
+              
+                <h1>Where Work Happens</h1>
+                <p>
+                  When your team needs to kick off a project, hire a new employee, deploy
+                  some code, review a sales contract, finalize next year's budget, measure
+                  an A/B test, plan your next office opening, and more, Slack has you
+                  covered.
+                </p>
+              
+            </div>            
+            
+            <div class="row">
+              <p class="col-md-4 email">Email address</p>
+              <button type="button" name="button" class="col-md-4 purple">GET STARTED</button>
+            </div>
+
+            <p class="question">Already using Slack? <a href="#">Sign in</a></p>
+
+          </div>          
+          <div class="col-lg-6 ">
```

Comment:
```suggestion
          <div class="col-lg-6 order-lg-first">
```
De esta manera entiende que cuando esté en large versión, tiene que coger la posición más a la izquierda, la cual toma como la primera, te dejo un [link](https://stackoverflow.com/questions/50745969/re-ordering-a-bootstrap-4-grid) de stack-overflow que me parece interesante.

Reply:
Perfecto, ya me ha quedado claro, gracias !